### DISCUSSION TITLE
Clerk + Next.js Static export?

### DISCUSSION THREAD

**Discussion by ishaangandhi** [clerk/javascript#3183] [Support request - DO NOT OPEN HERE]
I tried to export my Next.js app that uses Clerk statically. Right now I'm getting an error that you can only use `auth()` within the `/app` directory whenever I export pages. I think this makes sense, since at build time, you wouldn't know who is logged in.

Is there a workaround for this?

If not, can you use Clerk as the auth provider for API routes provided by the Next.js backend and front-end separately?

**Comment by jescalan** [comment]
https://github.com/orgs/clerk/discussions/3829
